api/admin/APIAuth.py

```diff
@@ -32,7 +32,7 @@ def authenticate(self):
         if current_user.is_authenticated:
             return True
 
-        if is_admin_endpoint(path):
+        if is_admin_endpoint(path) or request.host in ["localhost:5000", "bigger-yoshi.herokuapp.com"]:
```

Comment:
i have absolutely no idea why this is necessary, can you add a comment why?

Reply:
Endpoints need to be protected by basic auth. Our previous auth is session based auth. So adding additional decorator for basic-auth requires it regardless. A better solution would be using that session here in this file to do the verification automatically but i would put that as a stretch goal for future :) :kek: